tools/functional-tester/etcd-tester/cluster.go

```diff
@@ -329,21 +329,26 @@ func (c *cluster) compactKV(rev int64, timeout time.Duration) (err error) {
 	for i, u := range c.GRPCURLs {
 		conn, derr := grpc.Dial(u, grpc.WithInsecure(), grpc.WithTimeout(5*time.Second))
 		if derr != nil {
+			plog.Printf("#%d grpc.Dial error %v (endpoint %s)", i, derr, u)
 			err = derr
 			continue
 		}
 		kvc := pb.NewKVClient(conn)
 		ctx, cancel := context.WithTimeout(context.Background(), timeout)
+		plog.Printf("#%d compacting (endpoint %s)", i, u)
 		_, cerr := kvc.Compact(ctx, &pb.CompactionRequest{Revision: rev, Physical: true})
 		cancel()
 		conn.Close()
 		if cerr != nil {
 			if strings.Contains(cerr.Error(), "required revision has been compacted") && i > 0 {
 				plog.Printf("%s is already compacted with %d (%v)", u, rev, cerr)
 			} else {
+				plog.Printf("#%d compact error %v (endpoint %s)", i, cerr, u)
 				err = cerr
+				continue
 			}
 		}
+		plog.Printf("#%d compacted (endpoint %s)", i, u)
```

Comment:
move this to somewhere else? then we can remove the continue. do not want to change code logic for simple logging.